src/Debugging-Core/InstructionStream.extension.st

```diff
@@ -253,37 +258,20 @@ InstructionStream >> interpretV3PlusClosureJumpIfCond [
 
 { #category : #'*Debugging-Core' }
 InstructionStream >> nextByte [
-	"Answer the next bytecode."
+	"Answer the next bytecode"
+	self flag: 'Should this method be renamed nextBytecode? Is it possible thta it returns only a byte?'.
 
 	^self method at: pc
```

Comment:
What is the difference with followingByte? I was thinking this one advanced the stream, but it does not

Reply:
Indeed this is what we faced. nextByte is the nextByteCode while followingByte is the following byte of the byte code

Reply:
to me the current definition is super bad

```
followingByte
	"Answer the next bytecode."

	^self method at: pc + 1
```